packages/hello-gsm-admin/src/pages/index.tsx

```diff
@@ -5,7 +5,7 @@ import { MainPage } from 'PageContainer';
 import { ApplicantsType } from 'Types/application';
 import application from 'Api/application';
 import auth from 'Api/auth';
-import HeaderType from 'Types/header';
+import { HeaderType } from 'Types/header';
```

Comment:
```suggestion
import HeaderType from 'Types/header';
```

이 곳 import가 수정이 안된 듯 하네요